cool.py

```diff
@@ -545,7 +545,18 @@ def append_results(results_df):
     )
 
     job = BIGQUERY_CLIENT.load_table_from_dataframe(results_df, table_id, job_config=job_config, location="US")
-    job.result()  # Waits for table load to complete.
+   
+    try:
+        job_result = job.result()  # Waits for table load to complete.
+    except Exception as ex:
+        logging.error("unable to append rows into the results table! %s", ex)
```

Comment:
**question**: Do we need to log more information here to sort out any issues? Are your thoughts that the data will get picked up and processed later and not be an issue?

Reply:
Could we use these properties on the job result variable? 

https://github.com/googleapis/python-bigquery/blob/075aa66a7488636606d1ac0fe21690a37904746a/google/cloud/bigquery/job/base.py#L579-L599

Reply:
I added some more error info and check the job status to ensure success before running `update_index`